qscript.template

```diff
@@ -128,23 +142,135 @@ for layer in ${layers[@]}; do
             if [ -e $file ]; then
                numMsg=$(grep WarnElev $file | wc -l)
                if [ $numMsg = 0 ]; then
-                  echo "$THIS: No numerical instability detected in '$file' after completion of $jobtype.$layer job ${%JOBID%}." 2>&1 | awk -v level=INFO -v this=$THIS -f $SCRIPTDIR/monitoring/timestamp.awk | tee --append $SCENARIOLOG
+                  echo "cycle $CYCLE: $SCENARIO: No numerical instability detected in '$file' after completion of '$jobtype.$layer' job '${%JOBID%}'." 2>&1 | awk -v level=INFO -v this=$THIS -f $SCRIPTDIR/monitoring/timestamp.awk | tee --append $SCENARIOLOG
                else
                   ERROVALUE=1
-                  ERROMSG="$ERROMSG Detected $numMsg numerical instability messages in '$file' after completion of $jobtype.$layer job ${%JOBID%}."
+                  ERROMSG="$ERROMSG Detected '$numMsg' numerical instability messages in '$file' after completion of '$jobtype.$layer' job '${%JOBID%}'. "
                   RUNSUFFIX="error"
                fi
             fi
          done
```

Comment:
This is awesome. Would it be too much to as to put this check into a seperate script? 2 reasons:

1. this is obviously generally useful (and I'd like to use it with `asgs-mon` in some way)
2. at some point I'd really like to redo the queue scripts so they use Template::Toolkit

Reply:
Yes I think so, I will have a look to see how hard this might be.

Reply:
I've removed this from the queue script template and placed in a separate script in `util/output`.